aea/package_manager/base.py

```diff
@@ -65,7 +67,14 @@ def load_configuration(
     return cast(PackageConfiguration, configuration_obj)
 
 
-class BasePackageManager:
+class DepedencyMismatchErrors(Enum):
+    """Dependency mismatch errors."""
+
+    HASH_NOT_FOUND: int = 1
+    HASH_DOES_NOT_MATCH: int = 2
+
+
```

Comment:
Do we now have duplicate logic in `aea hash all` and here? It would be better to deduplicate?

Reply:
I've added deprecation warning for `aea hash all` command, this command does not make sense anymore so we will use `aea packages lock` to lock and perform dependency hash checks